app/controllers/Subscriptions.scala

```diff
@@ -132,9 +131,14 @@ class Subscriptions @Inject() (
       def handleInvalidForm(formWithErrors: Form[ChangeSubscriptionForm]) = Future.successful(BadRequest(errorHandler.badRequestTemplate))
 
       ChangeSubscriptionForm.form.bindFromRequest.fold(handleInvalidForm, handleValidForm);
-    }   
-  
-  def requestChangeApiSubscription(applicationId: ApplicationId, apiName: String, apiContext: ApiContext, apiVersion: ApiVersion, redirectTo: String, call: Call): ApplicationRequest[AnyContent] => Future[Result] = 
+    }
+
+  def requestChangeApiSubscription(applicationId: ApplicationId,
+                                   apiName: String,
+                                   apiContext: ApiContext,
+                                   apiVersion: ApiVersion,
+                                   redirectTo: String,
+                                   call: Call): ApplicationRequest[AnyContent] => Future[Result] =
     (request: ApplicationRequest[AnyContent]) => {
```

Comment:
Don't duplicate the typed decl.

Reply:
I presume by this that you mean the `implicit val r = request` line? Removing it means that we have to manually specify all the `implicit`s required by the view - are we ok with that, or is there a better way?